nixos/modules/services/web-apps/gotosocial.nix

```diff
@@ -0,0 +1,176 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.gotosocial;
+  settingsFormat = pkgs.formats.yaml { };
+  defaultSettings = {
+    application-name = "gotosocial";
+
+    host = "localhost:8080";
+    protocol = "http";
+
+    bind-address = "127.0.0.1";
+    port = 8080;
+
+    storage-local-base-path = "/var/lib/gotosocial/storage";
+
+    db-type = "sqlite";
+    db-address = "/var/lib/gotosocial/database.sqlite";
+  };
+  gotosocial-admin = pkgs.writeShellScriptBin "gotosocial-admin" ''
+    exec systemd-run \
+      -u gotosocial-admin.service \
+      -p DynamicUser=yes \
```

Comment:
Does dynamic user scale with lots of files? I think it is questionable to use it here

Reply:
Removed